lib/classes/Swift/IoException.php

```diff
@@ -19,9 +19,10 @@ class Swift_IoException extends Swift_SwiftException
      * Create a new IoException with $message.
      *
      * @param string $message
+     * @param int    $code
      */
-    public function __construct($message)
+    public function __construct($message, $code = 0)
```

Comment:
Actually, what about adding the full signature of \Exception by adding the previous exception as well?

```php
__construct($message = null, $code = 0, Exception $previous = null)
```